test/gleam/dynamic_test.gleam

```diff
@@ -1504,3 +1504,21 @@ pub fn decode9_test() {
     ]),
   )
 }
+
+type ClassifyAtom {
+  ClassifyAtom
+}
+
+pub fn classify_test() {
+  dynamic.from(True)
+  |> dynamic.classify
+  |> should.equal("Bool")
+
+  dynamic.from(False)
+  |> dynamic.classify
+  |> should.equal("Bool")
+
+  dynamic.from(ClassifyAtom)
+  |> dynamic.classify
+  |> should.equal("Atom")
```

Comment:
```suggestion
```

This failed on the JS target- atoms only exist on Erlang.

Reply:
Good to know, I though that the tests run only on Erlang. 
And it seems like you forgot to remove `ClassifyAtom` type...